.changeset/dirty-crews-invite.md

```diff
@@ -0,0 +1,5 @@
+---
+"@neo4j/graphql": patch
+---
+
+feat: add additional interface tests to verify the ordering of clauses
```

Comment:
mmm, I don't think that new tests should be part of the changelog :thinking:

Reply:
Good one! Will remove the changelog